### PR TITLE
Fix flaky personalization profile test and upgrade Cypress to 6 3.0

### PR DESCRIPTION
**Pull request by ddzz**
## Description
This PR fixes two minor issues in a Cypress test caused by overlapping elements. Cypress is also being upgraded from `6.2.1` to `6.3.0` in this PR.

## Testing done
- Test was run 60 times locally and on CI without any failures
- No regressions observed locally or on CI from the Cypress upgrade

## Acceptance criteria
- [ ] Tests pass locally and on CI

### CODE REVIEW

**Review by sanlouise — APPROVED** [review]

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ const checkModals = options => {
 
   // Make an edit
   cy.get(`#${editLineId}`)
-    .click()
-    .type('test');
+    .click({ force: true })
+    .type('test', { force: true });
```
Comment (erikphansen) [inline src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:41]: Just curious if `force` is actually needed for `type()` to work. Seems like once the click happens, typing would work.

**Review by erikphansen — APPROVED** [review]

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js (hunk 1 above)
Comment (ddzz) [reply src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:41]: @erikphansen The force is indeed needed. I thought adding the `force` to the `click()`would be sufficient, but the `type()` started failing after that. Seems like there's a minor rendering race condition that causes one or the other to fail, so we need the `force` in both spots.

**Review by 1Copenut — APPROVED** [review]
👍 LGTM!

File: src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js (hunk 1 above)
Comment (erikphansen) [reply src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js:41]: Thanks for confirming, @ddzz !

**Review by timwright12 — APPROVED** [review]

**Review by pjhill — APPROVED** [review]
Excellent -- thanks for getting this upgrade in quickly!

**Pull request merged**